components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/index.html

```diff
@@ -2746,6 +2761,9 @@ <h4 class='modal-title export-dialog-title' id='initialHeading'>Exporting Siddhi
     </div>
 </div>
 
+!-- Async API Dialog -->
```

Comment:
Remove this

Reply:
fixed with https://github.com/wso2/carbon-analytics/pull/1913/commits/d807fbc380a7203f1238bcad5425d84778a91878